.github/actions/spell-check/dictionary/colors.txt

```diff
@@ -0,0 +1,782 @@
+snow
+ghost-white
```

Comment:
this file only needs to cover the final short list of names once we compress it and delete the spaces, right?

Reply:
Yeah.. but the compressed names are hard to read, you know. I thought why not leaving the actual names also. It does not hurt anyone, right? It has `snow`, for crying out lot. That's definitely a dup.